app/views/posts/index.html.erb

```diff
@@ -17,9 +17,10 @@
     <div class="img-div"><%= image_tag post.image, width:"500px", class:"img-fluid" if post.image.attached?%></div>
     <%end%>
   </div>
-  <div class="card-footer text-muted">
-    <a><i class="fa fa-thumbs-up"> 7 </i></a></a>
-    <a <i class="fa fa-comment-o"> <%= link_to " comments", new_post_path %> </i></a>
+  <div class="card-footer text-muted like-and-comments">
+    <%= link_to '', like_path(post), class:'like fa fa-thumbs-up'  %>
+    <i><%= post.likes.count%></i>
```

Comment:
- [ ] **[OPTIONAL]** `post.likes.count` will perform queries to the database as it will count the number of likes columns for every post. As your application grows this will slow down your application so you need another mechanism to count the likes, here comes the `counter_cache` which basically cache the `count` and save it into separate column in `posts` table.
[1](https://blog.appsignal.com/2018/06/19/activerecords-counter-cache.html) [2](https://stackoverflow.com/questions/56913688/how-to-get-the-size-of-belong-to-record/56916353#56916353)